### PR TITLE
feat: adding some dynamism in the webchat

### PR DESCRIPTION
**Pull request by imOscarCrespo**
I added two new packages. https://www.framer.com/motion/ and https://www.react-reveal.com/ in order to add dynamism.

### CODE REVIEW

**Comment by imOscarCrespo** [comment]
after rebasing with master, i've to fix webchat component

File: packages/botonic-react/src/webchat/webchat.jsx [hunk 1]
```diff
@@ -28,6 +28,8 @@ import EmojiPicker from 'emoji-picker-react'
 import LogoMenu from '../assets/menuButton.svg'
 import LogoEmoji from '../assets/emojiButton.svg'
 import { MIME_WHITELIST } from '../constants'
+import { CUSTOM_WEBCHAT_PROPERTIES, MIME_WHITELIST } from '../constants'
+import { motion, useViewportScroll, useTransform } from 'framer-motion'
```
Comment (vanbasten17) [inline packages/botonic-react/src/webchat/webchat.jsx]: remove unused imports of code

File: packages/botonic-react/src/components/message.jsx [hunk 2]
```diff
@@ -6,6 +6,9 @@ import { WebchatContext, RequestContext } from '../contexts'
 import { Button } from './button'
 import { Reply } from './reply'
 import Logo from '../assets/botonic_react_logo100x100.png'
+import Fade from 'react-reveal/Fade'
+import LightSpeed from 'react-reveal/LightSpeed'
+import Slide from 'react-reveal/Slide'
```
Comment (vanbasten17) [inline packages/botonic-react/src/components/message.jsx]: remove unused

File: packages/botonic-react/src/components/message.jsx [hunk 3]
```diff
@@ -196,7 +199,43 @@ export const Message = props => {
                 borderLeft: 0,
                 marginLeft: -pointerSize + 1,
               }}
-            />
+              {...otherProps}
+            >
+              <div
```
Comment (vanbasten17) [inline packages/botonic-react/src/components/message.jsx]: if you had conflicts previously with this PR, maybe you can put it into the description of the PR if this is pending

**Comment by dpinol** [comment]
@imOscarCrespo please review if there are pending changes to do and merge

**Comment by vanbasten17** [comment]
closing this PR as I have already included the changes in https://github.com/hubtype/botonic/pull/514

**Pull request closed without merging**